tests/js/spec/integrations/phabricator.spec.jsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
```

Comment:
i do find it a bit weird these are in core and not in the sentry-plugins repo

Reply:
It's a bit weird but I think it makes sense since we want to make sure core doesn't break compatibility